webpack/assets/javascripts/react_app/components/PermissionDenied/PermissionDenied.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import PermissionDenied from '.';
+import Story from '../../../../../stories/components/Story';
+
+storiesOf('PermissionDenied', module)
```

Comment:
Now it's on global level, I would rather see it in the `Components/PermissionDenied`.